source/renderer/app/containers/staking/dialogs/DelegationSetupWizardDialogContainer.js

```diff
@@ -142,6 +144,15 @@ export default class DelegationSetupWizardDialogContainer extends Component<
     );
     const selectedPool = find(stakePools, pool => pool.id === selectedPoolId);
 
+    const getPledgeAddressUrl = (pledge: string) =>
```

Comment:
@daniloprates here we need `pledgeAddress` as a prop.

Reply:
✅